src/skin/css/firstRun.css

```diff
@@ -788,6 +788,13 @@ body {
   background-image:linear-gradient(#DFDFE6, #DFDFE6 30%, #fff 30%);
   padding:4rem 0;
 }
+#pb-privacy-policy, #pb-privacy-policy a {
+  color:#707070;
+  font-family:"OpenSans-Light",sans-serif;
+  font-size:0.7rem;
```

Comment:
Maybe make the text a bit larger?

Reply:
I tried with 0.9, 1 seems a bit too large. Let me know how this looks.